packages/server/src/api/queueRoutes.ts

```diff
@@ -137,3 +118,43 @@ export const queueRoutes: ServerRoute[] = [
     },
   },
 ];
+
+// for some reason, JOI.allow(null) means the property has to exist, but can be null
```

Comment:
> for some reason, `property: JOI.allow(null)` means the property has to exist, but can be null

I would have thought that the two would be synonymous. this is annoying. thoughts?

Reply:
imo this can be fixed in a separate PR. I think it's fine for now as long as we remember this weirdness